debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -47,6 +56,8 @@
 @NotThreadSafe
 public class KafkaDatabaseHistory extends AbstractDatabaseHistory {
 
+    private static final Duration MESSAGE_RETENTION_IN_DAYS = Duration.ofDays(365 * 10);
```

Comment:
Spec requires infinite retention so maybe this should be Max INT rather than 10 years?

Reply:
We are promoting either very long or infinite retention - @gunnarmorling any preference?

Reply:
Yes, infinite would be preferable, if doable.